frontend/src/scenes/pipeline/PipelineApp.tsx

```diff
@@ -40,16 +41,17 @@ export function PipelineApp(params: { kind?: string; id?: string } = {}): JSX.El
     if (!Object.values(PipelineTabs).includes(kind)) {
         return <NotFound object="pipeline app" />
     }
-    const { currentTab } = useValues(pipelineAppLogic)
+    const { tabs, currentTab } = useValues(pipelineAppLogic)
 
     if (!id) {
         return <Spinner />
     }
 
-    const tabToContent: Record<PipelineAppTabs, JSX.Element> = {
+    const tabToContent: Partial<Record<PipelineAppTabs, JSX.Element>> = {
         [PipelineAppTabs.Configuration]: <PipelineAppConfiguration />,
         [PipelineAppTabs.Metrics]: <AppMetrics pluginConfigId={id as number} />,
         [PipelineAppTabs.Logs]: <PipelineAppLogs id={id} kind={kind} />,
+        [PipelineAppTabs.History]: <ActivityLog id={id} scope={ActivityScope.PLUGIN} />,
```

Comment:
This does the same thing as the current history tab for plugins i.e. display the activities for all plugins. Unsure if this behaviour is intended, but it was definitely surprising for me. I would have expected the logs for the current plugin config (and maybe the associated plugin). We don't have such an endpoint however.

Reply:
yes please would love if we did current plugin

Reply:
I'm not sure if we should have two - history at the top level showing everything together and then per plugin level showing only for this plugin